roiextractors/extractors/schnitzerextractor/cnmfesegmentationextractor.py

```diff
@@ -78,8 +82,37 @@ def roi_locations(self):
         return roi_location
 
     @staticmethod
-    def write_segmentation(segmentation_object, savepath):
-        raise NotImplementedError
+    def write_segmentation(segmentation_object, savepath, **kwargs):
+        plane_no=kwargs.get('plane_no',0)
+        filename = os.path.basename(savepath)
+        savepath_folder = os.path.join(os.path.dirname(savepath),f'Plane_{plane_no}')
+        savepath = os.path.join(savepath_folder,filename)
```

Comment:
@Saksham20 use 'pathlib' instead

Reply:
That would be fine but it's not necessary